the_debuginator.h

```diff
@@ -1090,6 +1091,18 @@ DebuginatorItem* debuginator_create_array_item(TheDebuginator* debuginator,
 	item->leaf.description = description == NULL ? "" : description;
 	debuginator__adjust_num_visible_children(item->parent, 1);
 
+	if (description && !item->is_folder) {
+		const char* description = item->leaf.description;
+		float description_width = debuginator->size.x - 50;
```

Comment:
Note to self: Expose all of these hard coded constants as overrideable defines.. :)